src/utils/themeResolver.ts

```diff
@@ -94,6 +99,11 @@ export const deepMerge = <T extends PlainObject, U extends PlainObject>(
         result[key] = deepMerge(value1, value2);
       } else {
         // Otherwise, prioritize obj1's value
+        if (value1 && value2) {
```

Comment:
Similar comment which I know is not your change. I'd think the theme.config takes precedence. We should confirm with Product what they want to have happen, assuming the spec doesn't say it already.